config/tutorials/en/play-audio-ruby.yml

```
@@ -0,0 +1,54 @@
+title: How to Play Audio into a Call with Ruby
+products:
+ - voice/voice-api
+description: A tutorial showing you how to build an app that will play audio into a PSTN call
+languages:
+ - ruby
+
+introduction:
+  title: Introduction to this tutorial
+  description: This tutorial shows you how to play audio into a phone call using Ruby
+  content: |
+    # Introduction
+    The [Vonage Voice API](/voice/voice-api/overview) is the easiest way to build high-quality voice applications in the Cloud. In this tutorial you will learn how to stream pre-recorded audio into both inbound and outbound calls using Ruby and the Vonage Voice API.
+
+    This tutorial provides step by step instructions showing you how to:
+
+    1. Set up a Vonage account
+    1. Create a Vonage Application
+    1. Configure a Vonage Application 
+    1. Create an Sinatra application
+    1. Add the Vonage Ruby SDK to it
+    1. Receive a call and play audio into it
+    1. Place a call with streamed audio played into it
```

```suggestion
    1. Place a call and stream audio into it
```
^^ A little bit clearer?